### PR TITLE
Added Teamspeak3 template

### PR DESCRIPTION
**Pull request by Coolguy3289**

### CODE REVIEW

File: teamspeak3/teamspeak3.json [hunk 1]
```diff
@@ -0,0 +1,75 @@
+{
+  "pufferd": {
+    "type": "Voice",
```
Comment (LordRalex) [inline teamspeak3/teamspeak3.json]: Prefer types to be lowercased

File: teamspeak3/teamspeak3.json [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    "install": {
+      "commands": [
+        {
+          "files": "http://dl.4players.de/ts/releases/3.1.1/teamspeak3-server_linux_amd64-3.1.1.tar.bz2",
+          "type": "download"
+        },
+        {
+          "commands": [
+            "tar -xvf teamspeak3-server_linux_amd64-3.1.1.tar.bz2",
+	    "touch teamspeak3-server_linux_amd64/.ts3server_license_accepted"
```
Comment (LordRalex) [inline teamspeak3/teamspeak3.json]: In general, when it comes to anything license related, we always require it to be set explicitly, as we do not want to get into issues where we auto-accept licenses for folk who don't know it's there.

File: teamspeak3/teamspeak3.json [hunk 3]
```diff
@@ -0,0 +1,75 @@
+{
+  "pufferd": {
+    "type": "Voice",
+    "display": "Teamspeak 3",
+    "install": {
+      "commands": [
+        {
+          "files": "http://dl.4players.de/ts/releases/3.1.1/teamspeak3-server_linux_amd64-3.1.1.tar.bz2",
```
Comment (LordRalex) [inline teamspeak3/teamspeak3.json]: Should be able to use a ${version} tag if they do it, refer to our Minecraft ones for example.

**Comment by Coolguy3289** [comment]
@LordRalex I've updated per your recommendations.

**Comment by Coolguy3289** [comment]
I've also added a bit of context to the Arma Readme.

**Comment by Coolguy3289** [comment]
@LordRalex This is as good as I can get it for now. Can you merge? Thanks!

File: arma3/arma3.json [hunk 4]
```diff
@@ -25,12 +26,16 @@
...
+        "-config=${config}",
+        "-world=${world}",
+        "-mod=${mods}"
+	"-config=config_server.cfg"
       ],
       "program": "./arma3server"
     },
     "environment": {
-      "type": "tty"
+      "type": "standard"
```
Comment (LordRalex) [inline arma3/arma3.json]: srcds used tty, are you sure using standard will work here?

**Pull request merged**